circle.yml

```diff
@@ -1,22 +1,10 @@
 ---
-machine:
-  hosts:
-    dm.local: 127.0.0.1
-  environment:
-    BEET_HOME: /beetbox
-    BEET_BASE: /var/beetbox
-    BEET_USER: ubuntu
-    BEET_DEPENDENCIES: https://raw.githubusercontent.com/beetboxvm/beetbox/master/provisioning/beetbox.sh
-    DRUPAL_ROOT: /var/beetbox/docroot
-    DRUPAL_URL: "http://dm.local"
 dependencies:
   override:
-    - sudo cp ~/$CIRCLE_PROJECT_REPONAME/.beetbox/config.yml ~/vagrant.config.yml
-    - sudo cp -rf ~/$CIRCLE_PROJECT_REPONAME $BEET_BASE ; sudo chown -R $BEET_USER:$BEET_USER $BEET_BASE
-    - bash <(curl -s $BEET_DEPENDENCIES)
+    - bash <(curl -fsSL http://bit.ly/beetbuild)
 test:
   override:
     # Lint code.
-    - cd $BEET_BASE ; composer install ; vendor/bin/dcr install ; . ~/.profile ; dcr
+    - composer install ; vendor/bin/dcr install ; . ~/.profile ; dcr
```

Comment:
I would recommend running `composer install` as a `dependencies override` as more that `dcr` may need to be installed

Also, I recommend using `&&` to make sure that `dcr` gets installed properly (';' does not validate proper exit of previously executed command).
`vendor/bin/dcr install && . ~/.profile && dcr`